importer/MtDe.py

```diff
@@ -25,30 +33,36 @@ def update_labels(self):
             "en": self.name_en,
             "mt": self.name_mt
         }
-        for language in labels:
-            self.add_label(language, labels[language])
+        for lang, label in labels.items():
+            self.add_label(lang, label)
 
     def update_descriptions(self):
         """
-        Set the descriptions in English.
+        Set the descriptions in en and mt.
 
         Try to add the place name. Since the municipality
         names are titles of articles, sometimes they
         have the form of "$municipality (Malta)", in which
         case remove the bracketed part.
         """
+        patterns = {
+            "en": "cultural property in {}",
+            "mt": "propjeta' kulturali ġewwa {}"
+        }
         placename = "Malta"
-        description = "cultural property in {}"
-        if self.has_non_empty_attribute("gemeinde"):
-            municip_base = utils.get_rid_of_brackets(self.gemeinde)
-            placename = "{}, {}".format(municip_base, "Malta")
-        self.add_description("en", description.format(placename))
+        for lg, descr in patterns.items():
+            if self.has_non_empty_attribute("gemeinde"):
```

Comment:
_Note in case you update this anyway after #85 is merged, otherwise you can leave as is._

The if can be moved outside of the loop since placename is the same for both.

Reply:
Yes, as I commented above this will be influenced by #85 so it's waiting for that one.